src/main/java/com/maddyhome/idea/vim/extension/surround/VimSurroundExtension.kt

```diff
@@ -181,21 +181,25 @@ internal class VimSurroundExtension : VimExtension {
           }
 
           val registerValue = getRegisterForCaret(REGISTER, it.caret)
-          val innerValue = if (registerValue.isNullOrEmpty()) null else registerValue
+          val innerValue = if (registerValue.isNullOrEmpty()) emptyList() else registerValue
```

Comment:
Inspired from removed code [0] - set the `innerValue` as `emptyList()` - this will play nicely in line 198 when we'll convert it to string[1], for those cases when we deal with empty parenthesis.

[0] https://github.com/JetBrains/ideavim/pull/1007/files#r1810373587

[1] https://github.com/JetBrains/ideavim/pull/1007/files#diff-beac7e0e821f40c90568283d4d7ac4ab37de7362fab924aa616f94220fa85173R198